opennms-config-model/src/main/java/org/opennms/netmgt/config/discovery/ExcludeRange.java

```diff
@@ -98,14 +116,20 @@ public boolean equals(final Object obj) {
         if (obj instanceof ExcludeRange) {
             final ExcludeRange temp = (ExcludeRange)obj;
             return Objects.equals(temp.m_begin, m_begin)
-                    && Objects.equals(temp.m_end, m_end);
+                    && Objects.equals(temp.m_end, m_end)
+                    && Objects.equals(temp.m_location, m_location);
         }
         return false;
     }
 
     @Override
     public String toString() {
-        return "ExcludeRange [begin=" + m_begin + ", end=" + m_end + "]";
+        return "ExcludeRange{" +
```

Comment:
We normally don't include the `m_` prefix in the toString labels. While we're changing this, consider using `MoreObjects.ToStringHelper`.